### DISCUSSION TITLE
How to enable extra keys for Termux Float

### DISCUSSION THREAD

**Discussion by dipayand** [termux/termux-app#2523] [Q&A] [answered]
The release document for Version 0.118.0 states termux.float.properties can be setup to add extra keys for popup. Is there any help manual?

**Comment by agnostic-apollo** [comment, marked as answer]
Where does it say that? It only says

>Add support for `~/.termux/termux.float.properties`.

Extra keys are currently not supported. Check https://github.com/termux/termux-float#terminal-and-app-settings for what is supported.

**Reply by dipayand** [reply]
so what is allowed in termux.float.properties? shall I use another discussion thread for this?

**Reply by agnostic-apollo** [reply]
Re-read my comment.

**Reply by dipayand** [reply]
so sorry... please mark this as answered..
